models/issues/issue.go

```diff
@@ -1285,6 +1285,9 @@ func (opts *IssuesOptions) setupSessionNoLimit(sess *xorm.Session) {
 
 	if opts.AssigneeID > 0 {
 		applyAssigneeCondition(sess, opts.AssigneeID)
+	} else if opts.AssigneeID == -1 {
+		sess.NotIn("issue.id", builder.Select("issue.id").From("issue").Join("INNER", "issue_assignees", "issue.id = issue_assignees.issue_id").
```

Comment:
Maybe
```go
sess.And(builder.NotExists(
      builder.Select("id").From("issue_assignees").
      Where(builder.Expr("issue_assignees.issue_id = issue.id"),
))
```

Reply:
This modify has error: `Issues: unable to query Issues: Unsupported condition type`